bom/pom.xml

```diff
@@ -40,7 +40,7 @@ THE SOFTWARE.
   <properties>
     <asm.version>9.3</asm.version>
     <slf4jVersion>1.7.36</slf4jVersion>
-    <stapler.version>1711.v5b_1b_03f0fcf2</stapler.version>
+    <stapler.version>1733.v981fdd98007e</stapler.version>
```

Comment:
https://github.com/jenkinsci/stapler/compare/1711.v5b_1b_03f0fcf2...1733.v981fdd98007e contains a number of other changes, of which https://github.com/jenkinsci/stapler/pull/397 could conceivably introduce a regression (https://github.com/jenkinsci/stapler/pull/396 less likely). Seems safer to backport only https://github.com/jenkinsci/stapler/pull/402 right?

Reply:
Possibly. I think this is good enough.

Reply:
I will backport only jenkinsci/stapler#402.

And you are welcome, Jesse!

Reply:
Backported only the needed changes. Updated review description. Re-did testing.